components/Alert.js

```diff
@@ -27,7 +27,7 @@ export default function Alert({ type, message }) {
     case "warning":
       return (
         <div
-          className={`alert-${type} mt-3 bg-yellow-100 rounded-lg py-5 px-6 mb-3 text-base text-yellow-700 inline-flex items-center w-full`}
+          className={`alert-${type}  bg-yellow-100 rounded-lg py-5 px-6 mb-3   text-base text-yellow-700 inline-flex items-center w-full`}
```

Comment:
If we remove the margin top, the component will snap with the element above so instead use my property

Reply:
After cloning the project, when attempting to run it using npm run dev, I received the following error message: 

![image](https://github.com/EddieHubCommunity/BioDrop/assets/81041326/943ef18b-5948-46f3-853d-381b5108de2c)

As a result of this issue, I proceeded to install the Node.js package by running the "npm install"
I would appreciate it if you could provide guidance on how to run it in my terminal without the need to download any additional dependencies.

"If we remove the margin top, the component will snap with the element above so instead use my property" I have made this adjustment.

Reply:
run ```npm ci ``` instead of ```npm install```. Eddie had made a video on it you can check that out for reference.
https://www.youtube.com/watch?v=jHkIXYVpMTo&ab_channel=EddieJaoude .You can ask question on our discord server for any kind of help.

Reply:
Thank you for the video, I was able to figure it out